aspnetcore/signalr/authn-and-authz/sample/wwwroot/js/chat.ts

```diff
@@ -34,27 +33,18 @@ class ViewModel {
     public connection: signalR.HubConnection;
     public connectionStarted: boolean;
 
-    public async loginFormSubmitted(evt: Event) {
+    // TODO: Bind this method to a UI event that is triggered when the user is logging in
```

Comment:
Did you mean to leave this as a TODO?

Reply:
Nvm, you clearly did.

Reply:
Yep :). It's unfortunate that the sample isn't runnable, but when you run it it gives you a pretty clear message to go look at these TODOs and hook things up.